dts/arm/nordic/nrf9160_common.dtsi

```diff
@@ -152,6 +145,38 @@ spi3: spi@b000 {
 	label = "SPI_3";
 };
 
+pwm0: pwm@21000 {
```

Comment:
This has only be added for nRF9160, right?

Reply:
I am not sure if I correctly understood the question.
These nodes were missing for nRF9160. nRF5 family has them already defined (and all the other stuff, like bindings, is already in place).

Reply:
Ah right - sorry, I did not check thoroughly.